examples_gbs/run_tutorial_points.py

```diff
@@ -0,0 +1,137 @@
+#
+# Let's construct the RBF kernel with the parameter :math:`\sigma` set to 1.0.
+
+K = points.kernel(R, 1.0)
+
+##############################################################################
+# We generate 10 samples with an average number of 40 points per sample by calling
+# the :func:`~.points.sample` function of the :mod:`~.gbs.points` module.
+
+samples = points.sample(K, 40.0, 10)
+
+##############################################################################
+# We visualize the first sample by using the :func:`~.plot_points` function of
+# the :mod:`~.gbs.plot` module. The point patterns generated by the permanental point process
+# typically have a higher degree of clustering compared to a uniformly random pattern.
+
+plotly.offline.plot(
+    plot.plot_points(R, samples[0], point_size=10), filename="Points.html"
+)
+
+##############################################################################
+# .. raw:: html
+#     :file: ../../examples_gbs/Points.html
+
+##############################################################################
+# Outlier Detection
+# -----------------
+#
+# When the distribution of points in a given space is inhomogeneous, GBS point processes typically
+# sample points from the dense regions. In a data set containing a uniform grid of points and
+# some dense clusters, this feature of the GBS point processes can be used to detect the
+# high-density points that can be considered as outliers with respect to the homogeneous
+# background. In this example, we create two dense clusters and place them in a two-dimensional
+# space containing evenly-distributed points. We show that the permanental point process samples
+# points from the dense clusters with a higher probability.
+#
+# We first create the data points. The clusters have 20 points each and the points have a
+# standard deviation of 1 and are centered at [x = 5, y = 5] and [x = 15, y = 15], respectively.
+# The grid points are arranged in a 20 :math:`\times` 20 square pattern.
+
+grid = np.array([(i, j) for i in range(20) for j in range(20)])
+
+clusters = make_blobs(n_samples=40, centers=[[5, 5], [15, 15]], cluster_std=1)[0]
+
+R = np.concatenate((clusters, grid))
+
+##############################################################################
+# Then construct the kernel matrix and generate 500 samples.
+
+K = points.kernel(R, 1.0)
+
+samples = points.sample(K, 40.0, 500)
+
+##############################################################################
+# We obtain the indices of 40 points that appear most frequently in the permanental point
+# process samples and then visualize them. The majority of the commonly appearing points belong
+# to the dense clusters.
+
+gbs_frequent_points = np.argsort(np.sum(samples, axis=0))[-40:]
+
+plotly.offline.plot(
+    plot.plot_points(
+        R,
+        [1 if i in gbs_frequent_points else 0 for i in range(len(samples[0]))],
+        point_size=10,
+    ),
+    filename="Outliers.html",
+)
+
```

Comment:
I feel like the tutorial ends too abruptly. It would be good to include a concluding paragraph, maybe outlining other ways in which point processes can be used and encouraging the user to explore them.

Reply:
For example, it would be good to mention that the real challenge happens in high-dimensional spaces and GBS can potentially help there too